packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -30,10 +35,32 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
     return getSelectedItemsFromValue<T>(value, options);
   }, [value, options, isAsync]);
 
-  const multiStyles = useStyles2(getMultiComboboxStyles);
-
   const [items, _baseSetItems] = useState(isAsync ? [] : options);
   const [isOpen, setIsOpen] = useState(false);
+  const [shownItems, setShownItems] = useState<number>(selectedItems.length);
+  const [measureRef, { width }] = useMeasure<HTMLDivElement>();
```

Comment:
We don't have the `width` prop working at the moment, maybe we can use that as a base? If you have a set width it should straightforward, but in the other cases it needs to be measured.

Reply:
Added support for `width`, I think it works but didn't have time to overly test it 😅 If you could pull storybook and give it a run 🙏